src/main/java/com/imsweb/x12/Separators.java

```diff
@@ -88,6 +97,24 @@ public String toString() {
         return "[" + _segment + "," + _element + "," + _composite + "]";
     }
 
+    private void setSeparatorPatterns(Character segment, Character element, Character composite) {
+        _segmentPattern = Pattern.compile(Pattern.quote(segment.toString()));
+        _elementPattern = Pattern.compile(Pattern.quote(element.toString()));
+        _compositePattern = Pattern.compile(Pattern.quote(composite.toString()));
+    }
+
+    public String[] splitElement(String line) {
+        return (line != null && _elementPattern != null) ? _elementPattern.split(line) : null;
```

Comment:
This method returns null in certain cases, but none of the code that calls it checks for null so I assume there could be NullPointerExceptions.  For example:

```java
String[] elementsStr = _separators.splitElement(elementText);
Element[] elements = new Element[elementsStr.length - 1];
```